packages/contracts/scripts/write-constants.ts

```diff
@@ -10,6 +10,9 @@ import {
 } from '../src/addresses'
 import { SPHINX_NETWORKS, SPHINX_LOCAL_NETWORKS } from '../src'
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { version } = require('../package.json')
```

Comment:
nit: Can we use the standard import syntax here? i.e. `import { version } from '../package.json'`

Reply:
Yes, we can; not sure why I didn't just do that, tbh.